dist/smartbanner.js

```diff
@@ -428,7 +428,15 @@ var SmartBanner = function () {
       }
 
       var bannerDiv = document.createElement('div');
-      document.querySelector('body').appendChild(bannerDiv);
+      if (this.options.prependTarget !== undefined) {
+        var parent = this.options.prependTarget;
+        document.querySelector(parent).insertBefore(bannerDiv, parent.firstChild);
```

Comment:
I failed to get the prepending working properly. 

The test case:

__META__:
```html
  <meta name="smartbanner:prepend-target" content="#wrapper">
```

__HTML__:
```html
  <div id="wrapper">
    <div id="first"></div>
    <div id="last"></div>
  </div>
```

__Result__:
```html
<div id="wrapper">
    <div id="first"></div>
    <div id="last"></div>
  <div class="smartbanner smartbanner--ios js_smartbanner">
      <a href="javascript:void();" class="smartbanner__exit js_smartbanner__exit"></a>
      <div class="smartbanner__icon" style="background-image: url(http://a3.mzstatic.com/us/r30/Purple60/v4/c1/3b/b0/c13bb085-64c0-cc90-f97a-521b96963986/icon350x350.jpeg);"></div>
      <div class="smartbanner__info">
        <div>
          <div class="smartbanner__info__title">Smart Application</div>
          <div class="smartbanner__info__author">SmartBanner Contributors</div>
          <div class="smartbanner__info__price">FREE - On the App Store</div>
        </div>
      </div>
      <a href="https://itunes.apple.com/us/genre/ios/id36?mt=8" target="_blank" class="smartbanner__button"><span class="smartbanner__button__label">View</span></a>
    </div></div>
```

Reply:
I'm sorry, I must have had wrongly generated the minified file. I pushed the correct file and it should work properly.